sigle/src/modules/analytics/PublishedStoryItem.tsx

```diff
@@ -0,0 +1,116 @@
+import { ArrowRightIcon } from '@radix-ui/react-icons';
+import { stagger } from 'motion';
+import { useMotionAnimate } from 'motion-hooks';
+import { useEffect } from 'react';
+import { styled } from '../../stitches.config';
+import { SubsetStory } from '../../types';
+import { Box, Flex, Text } from '../../ui';
+
+const StoryImage = styled('img', {
+  objectFit: 'cover',
+  objectPosition: 'center',
+  width: 60,
+  height: 43,
+  br: '$1',
+  ml: '$2',
+});
+
+interface PublishedStoryItemProps {
+  story: SubsetStory;
+}
+
+export const PublishedStoryItem = ({ story }: PublishedStoryItemProps) => {
+  const { play } = useMotionAnimate(
+    '.story-item',
+    { opacity: 1 },
+    {
+      delay: stagger(0.075),
+      duration: 0.5,
+      easing: 'ease-in-out',
+    }
+  );
+
+  // Play the animation on mount of the component
+  useEffect(() => {
+    play();
+  }, []);
+
+  return (
+    <Flex
+      className="story-item"
+      align="center"
+      css={{
+        borderTop: '1px solid $colors$gray6',
+        height: 68,
+        cursor: 'pointer',
+        position: 'relative',
+        opacity: 0,
+
+        '& svg': {
+          transition: 'transform .3s ease-in-out',
+        },
+
+        '&:hover': {
+          backgroundColor: '$gray2',
+          '& svg': {
+            transform: 'translateX(-4px)',
+          },
+
+          '& div:first-child': {
+            display: 'block',
+            backgroundColor: '$gray11',
+          },
+        },
+
+        '& div:first-child': {
+          position: 'absolute',
+          width: '$1',
+          top: -1,
```

Comment:
styles here are ensuring indicator is positioned over top border and top border of element below:

![Screenshot 2022-05-05 at 12 40 36](https://user-images.githubusercontent.com/65421744/166916192-13bdc0bc-8df5-480e-8576-59bea13df691.png)